conbench/templates/benchmark-entity.html

```diff
@@ -30,6 +30,22 @@
 </div>
 <br>
 
+<div class="row">
+  <div class="col-md-12">
+    Click on a gray glyph in the plot to see corresponding benchmark run details.
```

Comment:
My inclination is that "glyph" is too jargon-y. What if one directly linked it to legend for common language?

```suggestion
    Click on any grey point (benchmark mean) in the plot to see corresponding benchmark run details.
```

Reply:
Thanks! I like that. And you also revealed your preference in the gray vs grey game :).